src/Integration.Vsix/ErrorList/IssuesSnapshot.cs

```diff
@@ -84,10 +84,14 @@ public override bool TryGetValue(int index, string keyName, out object content)
                     return true;
 
                 case StandardTableKeyNames.Line:
+                    // Note: the line and column numbers are taken from the SnapshotSpan, not the Issue.
+                    // The SnapshotSpan represents the live document, so the text positions could have
+                    // changed from those reported from the Issue.
                     content = this.issueMarkers[index].Span.Start.GetContainingLine().LineNumber;
                     return true;
 
                 case StandardTableKeyNames.Column:
+                    // Use the span, not the issue. See comment immediately above.
```

Comment:
It might be an overkill, but maybe it's worth creating a function for `this.issueMarkers[index].Span.Start.GetContainingLine()` and add a proper doc comment on it.

Reply:
It's a thought. I'm not going to make any code changes in this PR though.